plugins/node.d/amavis

```diff
@@ -18,11 +20,11 @@ The configuration environment variables are available
 
 =item amavislog
 
-Path to logfile (Default: "/var/log/mail/mail.info")
+Path to logfile (Default: "/var/log/maillog")
 
 =item logtail
 
-Path to logtail command (Default: "logtail")
+Path to logtail command (Default: "/usr/local/bin/logtail")
```

Comment:
I do not think, you should change this to a value that is only suitable for systems without a packaged `logtail`. At least here on my system it would work before (just using the system-wide configured PATH), but would fail with the new default.